api/src/user/user.controller.ts

```diff
@@ -9,13 +9,16 @@ import {
   Post,
   Put,
   Query,
+  UseGuards,
 } from '@nestjs/common';
 import { UserService } from './user.service';
+import { JwtAuthGuard } from '../auth/jwt-auth.guard';
 
 @Controller('user')
```

Comment:
Can't we add the UseGuards on the controller instead of adding it on each route? As we have to securise all the routes of the controller

Reply:
as upwards, cleaver 1